### PR TITLE
add version for pretty tables 

### PR DESCRIPTION
**Pull request by jd-lara**
I am not sure why the tests are failing in #80. I am opening this PR to check the failing tests locally.

### CODE REVIEW

**Comment by jd-lara** [comment]
@nickrobinson251 I also allowed for newer versions of DocStringExtensions since its blocking the update to 0.9.2

**Comment by jd-lara** [comment]
bump to @nickrobinson251

File: Project.toml [hunk 1]
```diff
@@ -12,10 +12,10 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 DataFrames = "1"
-DocStringExtensions = "0.8.5, 0.9"
+DocStringExtensions = "0.8.5, ~0.9"
```
Comment (nickrobinson251) [inline Project.toml:15]: isn't it the same thing, to specify `~0.9` and `0.9`?
https://pkgdocs.julialang.org/v1/compatibility/#compat-pre-1.0

**Review by nickrobinson251 — APPROVED** [review]
thanks! if CI uses v2 and is happy, then i'm happy

File: Project.toml (hunk 1 above)
Comment (nickrobinson251) [reply Project.toml:15]: i think it is, so i'll put this back for sake of simplicity
```suggestion
DocStringExtensions = "0.8.5, 0.9"
```

**Pull request merged**